tailwind.config.ts

```diff
@@ -7,7 +7,36 @@ export default {
     extend: {
       colors: {
         primary: '#1F2937',
+        onPrimary: '#F9FAFB',
```

Comment:
Camel case - super interesting! Creates a nice contrast to tailwind baked-in values that use `-`.